examples/extensions/add_geo_target.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Adds a geo target to an extension feed item for targeting."""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.api_core import protobuf_helpers
+
+
+def main(client, customer_id, feed_item_id, geo_target_constant_id):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID.
+        feed_item_id: the ID of an extension feed item.
+        geo_target_constant_id: the geo target constant ID to add to the
+            extension feed item.
+    """
+    extension_feed_item_service = client.get_service(
+        "ExtensionFeedItemService", version="v5"
+    )
+
+    extension_feed_item_op = client.get_type(
```

Comment:
Nit: almost all the examples use `operation` instead of `op`.

Reply:
SG updated.